build.gradle

```diff
@@ -179,6 +180,10 @@ subprojects {
                 "uk.gov.ida:saml-test:${saml_lib_version}",
         )
 
+        proxy_node_shared(
```

Comment:
Could add this config to `snyk/configurations.sh` so that bouncy caste is tested by snyk for vulnerabilites.

Reply:
Done 👍